kratos/geometries/line_2d_2.h

```diff
@@ -864,7 +864,7 @@ class Line2D2 : public Geometry<TPointType>
         const double tol = 1e-14; // Tolerance
 
         // Normal
-        array_1d<double,2> Normal = ZeroVector(2);
```

Comment:
You can avoid creating a temporary ublas vector by using the array_1d constructor:
`array_1d<double,2> Normal(2,0.0);`